### DISCUSSION TITLE
Datasheet for Badger Meter

### DISCUSSION THREAD

**Discussion by daelsc** [tronikos/esphome-magnetometer-water-gas-meter#27] [General]
Hello, I followed your guide and it works great!  I also have a Badger Meter (mine is model 70).  You mention the default value is for a Model 35; can you share where you found the datasheet for that meter?

**Comment by tronikos** [comment]
(FYI, I moved this to discussions since it isn't an issue.)
I couldn't find the datasheet. I got the value from the flume water meter I had before.

**Reply by tronikos** [reply]
I just found https://www.instrumart.com/assets/badger-er10_manual.pdf

<img width="713" alt="image" src="https://github.com/user-attachments/assets/dd816e64-aeca-4c69-bc81-b7b0674bf81e">

The value I got from flume is `0.01008156` which corresponds to `2/0.01008156 = 198.38` which almost matches the 25 RCDL model. But I have the Badge Meter Model 35. Maybe when I sent the photo to flume they misread it? FYI, when you setup the flume you have to send them a picture. I assume to assign the correct value to your account.

**Reply by tronikos** [reply]
I reached out to Flume support and my account is correctly associated with a Badge Meter Model 35. If the above datasheet is correct, maybe Flume have incorrectly associated model 35 with the pulses/gallon value of the model 25? I've never bothered to compare with the meter reading or with my bill. I mostly care about alerts and not accurate usage.

**Comment by aneisch** [comment]
Perhaps this applies? https://www.instrumart.com/assets/Badger-EPT1-datasheet.pdf

![Screenshot_20240720-065021](https://github.com/user-attachments/assets/107a7524-17ba-4b88-b8fa-d7c7df5afd7f)

**Reply by tronikos** [reply]
Not sure. The values don't seem to match mine.

**Comment by daelsc** [comment]
Great find on the datasheets.  Unfortunately, I can't make any of the numbers make sense with my Model 70 meter either.  Using a 4-cup measuring cup I calibrated mine to 0.0493. (Yes, I know using such a small volume isn't the most accurate way to calibrate but it's a quick and easy starting point...how accurate do I really need to be?)

**Reply by tronikos** [reply]
You aren't that far off the value from the datasheet. From the datasheet: `2/46.752 = 0.0427789`.

**Reply by daelsc** [reply]
wow, you're totally right.  I don't know why I thought that value didn't work.  I will give that a whirl.

**Reply by eigenphase** [reply]
I found another datasheet here: https://www.usabluebook.com/product/attachment/54253/542523.pdf
